src/Sylius/Bundle/CoreBundle/Tests/Responses/Expected/customer/index_response.json

```diff
@@ -0,0 +1,144 @@
+{
+    "page": 1,
+    "limit": 10,
+    "pages": 2,
+    "total": 11,
+    "_links": {
+        "self": {
+            "href": "\/api\/customers\/?page=1&limit=10"
+        },
+        "first": {
+            "href": "\/api\/customers\/?page=1&limit=10"
+        },
+        "last": {
+            "href": "\/api\/customers\/?page=2&limit=10"
+        },
+        "next": {
+            "href": "\/api\/customers\/?page=2&limit=10"
+        }
+    },
+    "_embedded": {
+        "items": [
+            {
+                "id": @integer@,
+                "email_canonical": "@string@.isEmail()",
+                "first_name": "@string@",
+                "last_name": "@string@",
+                "_links": {
+                    "user": {
+                        "href": "@string@.contains('\/api\/users\/')"
+                    }
+                },
+                "_embedded": {
+                    "user": {
+                        "id": @integer@,
+                        "username": "@string@.isEmail()",
+                        "enabled": false
```

Comment:
This field is random ([reference](https://github.com/Sylius/Sylius/pull/4037/files#diff-315af8203f9d8c1594c08221cd2b0397R5)), so this test will randomly fail.